lib/spack/spack/cmd/ci.py

```diff
@@ -38,6 +38,10 @@ def deindent(desc):
     return desc.replace("    ", "")
 
 
+def encode_path(path: str) -> str:
+    return path.encode("unicode-escape".decode())
```

Comment:
@johnwparent Am I missing something about these lines from the cherry-pick?

```
  File "C:\builds\spack\spack\lib\spack\spack\cmd\ci.py", line 584, in ci_rebuild
    [SPACK_COMMAND, "-e", encode_path(env.path), "bootstrap", "now"],
                          ^^^^^^^^^^^^^^^^^^^^^
  File "C:\builds\spack\spack\lib\spack\spack\cmd\ci.py", line 42, in encode_path
    return path.encode("unicode-escape".decode())
                       ^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'str' object has no attribute 'decode'. Did you mean: 'encode'?
```

Reply:
Yeah I missed the commit where I patch that typo, should be 8ff51a4fc50405be31b7b8bc6834f82a3ca16620